activity_browser/app/bwutils/commontasks.py

```diff
@@ -68,6 +68,15 @@ def get_databases_data(databases):
     yield data
 
 
+def get_editable_databases():
```

Comment:
There is a bug when a database is deleted. `user_project_settings` needs to be updated when a project is deleted, otherwise this function returns invalid dbs